packages/@sanity/desk-tool/src/pane/TypePane.js

```diff
@@ -33,7 +34,7 @@ export default withRouterHOC(class TypePane extends React.PureComponent {
     // const isActive = !selectedType && !action && !selectedDocumentId
 
     return (
-      <Pane {...this.props}>
+      <Pane {...this.props} styles={contentStylesOverride}>
```

Comment:
Is this intentional? Isn't `contentStylesOverride` a CSS modules-export - shouldn't it be a selector and passed to `classNames` instead of `styles`? Or am I missing something?

Reply:
It is intentional, because Pane is a styleable component.

Reply:
Of course. Got confused by the prop-name 🙃 .